doc/classes/FileDialog.xml

```diff
@@ -68,10 +68,10 @@
 			The currently selected file path of the file dialog.
 		</member>
 		<member name="filters" type="PoolStringArray" setter="set_filters" getter="get_filters">
-			Set file type filters. This example shows only .png and .gd files [code]set_filters(PoolStringArray(["*.png ; PNG Images","*.gd ; GD Script"]))[/code].
+			The available file type filters. For example, this shows only [code].png[/code] and [code].gd[/code] files: [code]set_filters(PoolStringArray(["*.png ; PNG Images","*.gd ; GDScript Files"]))[/code].
 		</member>
 		<member name="mode" type="int" setter="set_mode" getter="get_mode" enum="FileDialog.Mode">
-			Set dialog to open or save mode, changes selection behavior. See enum [code]Mode[/code] constants.
+			The dialog's open or save mode, which affects the selection behavior. See enum [code]Mode[/code] constants.
 		</member>
 		<member name="mode_overrides_title" type="bool" setter="set_mode_overrides_title" getter="is_mode_overriding_title">
 			If [code]true[/code], changing the [code]Mode[/code] property will set the window title accordingly (e.g. setting mode to [constant MODE_OPEN_FILE] will change the window title to "Open a File").
```

Comment:
I know you didn't touch this, but I may as well take note of what I think it could be changed to:

"If [code]true[/code], changing the **[member mode]** property will set the window title accordingly. **For example,** setting **[member mode]** to [constant MODE_OPEN_FILE] will change the window title to "Open a File"."